src/dope-node/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"dope-node/communication"
+	"flag"
+	"fmt"
+)
+
+func main() {
+	isBoostrapServer := flag.Bool("bootstrap", false, "Is this node running as a bootstrap")
+	bootstrapServerAddress := flag.String("bootstrap_address", "127.0.0.1:7312", "An IP address to the bootstrap server")
+	port := flag.Int("port", 7312, "Port to run websocket listener on")
```

Comment:
we'll be using [viper](https://github.com/spf13/viper) anyways, so we will refactor it when we move there